sources/packages/backend/apps/queue-consumers/src/processors/schedulers/student-application-notifications/student-application-notifications.scheduler.ts

```diff
@@ -39,7 +44,28 @@ export class StudentApplicationNotificationsScheduler extends BaseScheduler<void
         `Processing student application notifications job. Job id: ${job.id} and Job name: ${job.name}.`,
       );
 
-      // TODO: Get applications that have a disability status mismatch  and check PDPPD status
+      const eligibleApplications =
+        await this.applicationService.getEligibleApplicationsForNotification();
+
+      for (const application of eligibleApplications) {
+        const notification: StudentPDPPDNotification = {
+          userId: application.student.user.id,
+          givenNames: application.student.user.firstName,
+          lastName: application.student.user.lastName,
+          email: application.student.user.email,
+          applicationNumber: application.applicationNumber,
+        };
+
+        await this.notificationActionsService.saveStudentApplicationPDPPDNotification(
+          notification,
+          application.currentAssessment.id,
+        );
+      }
+
+      this.logger.log(JSON.stringify(eligibleApplications));
```

Comment:
Please remove this log. The `processSummary` will be logged to the console and to the job log in the finally block.